docs/dev-guide/golang/introduction.md

```diff
@@ -0,0 +1,136 @@
+---
+id: introduction
+title: Introduction to the Temporal Go SDK
+sidebar_label: Introduction
+sidebar_position: 1
+description: The Temporal Go SDK developer's guide provides a comprehensive overview of the structures, primitives, and features used in Temporal Application development.
+slug: /dev-guide/go/introduction
+toc_max_heading_level: 4
+keywords:
+- guide-context
+tags:
+- guide-context
+---
+
+<!-- THIS FILE IS GENERATED. DO NOT EDIT THIS FILE DIRECTLY -->
+
+Welcome to the Temporal Go SDK developer’s guide!
+
+:::success Temporal Go SDK API reference
+
+https://pkg.go.dev/go.temporal.io/sdk
+
+:::
+
+Temporal Python SDK first released on September 29, 2020, and has been enabling Python developers to build Temporal Applications ever since.
```

Comment:
This info about the Python SDK release date is not relevant, inaccurate, and seems to be copied and pasted in multiple locations.